Xamarin.PropertyEditing.Mac/Controls/Custom/BaseTextField.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using AppKit;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	public class BasePropertyControlTextField : NSTextField
```

Comment:
- Does this need to be public?
- Can we try a different name? It's not purely a base if we're using it directly.
- The file name really should match the class name

Reply:
Do you have any naming suggestions?
Only other name I can suggest would be one denoting functionality. 
So `EllipsisTextField` ?

Reply:
Could just be `PropertyTextField`

Reply:
Sure, why not.